api/classes/platform/process/tasks/ABProcessTaskServiceInsertRecord.js

```diff
@@ -267,10 +265,43 @@ module.exports = class InsertRecord extends InsertRecordTaskCore {
                   rawData
                );
                break;
+            case "6":
```

Comment:
If I understand the case here:  you can have a number of keys defined: key1, key2, key3, ...

And if ANY one of those keys return a value, you want to use that one, right?  

If >1 of those keys return a value ... which one do you use?  First one? Last one?


Also, looking at the flow of this loop, it looks like the **LAST** value is always the one used.  So if Key2 returns a valid value, but Key3 is ```null```,  ```null``` is what is returned.

Reply:
Thank you. 

I miss to implement to combine values in the field supports Many values.
I will do it.